int/api/html/front/website.js

```diff
@@ -171,36 +170,38 @@ async function tableInsert(resp, count) {
     const cell3 = row.insertCell();
     cell0.innerHTML = resp.name;
     cell1.innerHTML = addressInnerHTML(resp.address);
-    if (resp.brokenChunks != null) {
-        missingChunks[count] = await resp.brokenChunks;
-        cell2.innerHTML = `
-            <div class="broken-chunks">
-                <div class="tooltip">
-                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-exclamation-triangle" viewBox="0 0 16 16">
-                        <path d="M7.938 2.016A.13.13 0 0 1 8.002 2a.13.13 0 0 1 .063.016.146.146 0 0 1 .054.057l6.857 11.667c.036.06.035.124.002.183a.163.163 0 0 1-.054.06.116.116 0 0 1-.066.017H1.146a.115.115 0 0 1-.066-.017.163.163 0 0 1-.054-.06.176.176 0 0 1 .002-.183L7.884 2.073a.147.147 0 0 1 .054-.057zm1.044-.45a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767L8.982 1.566z"/>
-                        <path d="M7.002 12a1 1 0 1 1 2 0 1 1 0 0 1-2 0zM7.1 5.995a.905.905 0 1 1 1.8 0l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995z"/>
-                    </svg>
-                    <span class="tooltiptext">Website corrupted. Click on the gear wheel to fix it.</span>
-                </div>
-                <div class="repair-gear">
-                    <input id='repair-fileid${count}' type='file' hidden/>
-                    <svg id='fix-broken-website-${count}' xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-gear-fill" viewBox="0 0 16 16">
-                        <path d="M9.405 1.05c-.413-1.4-2.397-1.4-2.81 0l-.1.34a1.464 1.464 0 0 1-2.105.872l-.31-.17c-1.283-.698-2.686.705-1.987 1.987l.169.311c.446.82.023 1.841-.872 2.105l-.34.1c-1.4.413-1.4 2.397 0 2.81l.34.1a1.464 1.464 0 0 1 .872 2.105l-.17.31c-.698 1.283.705 2.686 1.987 1.987l.311-.169a1.464 1.464 0 0 1 2.105.872l.1.34c.413 1.4 2.397 1.4 2.81 0l.1-.34a1.464 1.464 0 0 1 2.105-.872l.31.17c1.283.698 2.686-.705 1.987-1.987l-.169-.311a1.464 1.464 0 0 1 .872-2.105l.34-.1c1.4-.413 1.4-2.397 0-2.81l-.34-.1a1.464 1.464 0 0 1-.872-2.105l.17-.31c.698-1.283-.705-2.686-1.987-1.987l-.311.169a1.464 1.464 0 0 1-2.105-.872l-.1-.34zM8 10.93a2.929 2.929 0 1 1 0-5.86 2.929 2.929 0 0 1 0 5.858z"/>
-                    </svg>
-                </div>
-            </div>`;
-        document
-            .getElementById(`fix-broken-website-${count}`)
-            .addEventListener("click", function () {
-                document.getElementById(`repair-fileid${count}`).value = null;
-                document.getElementById(`repair-fileid${count}`).click();
-            });
-        document.getElementById(`repair-fileid${count}`).addEventListener("change", function (evt) {
-            let files = evt.target.files;
-            nextFileToUpload = files[0];
-            onSubmitDeploy("uploadMissingChunks" + count);        
-        });
-    } else {
+    // TO BE UNCOMENTED : Because this was blocking for our tests at the end Uncoment this 
```

Comment:
Unclear.
What was the error and how to produce it?
What is the end you're talking about?

Reply:
when we fetch websites, if there is an issue to detect the missing chunk nothing will display so we don't know if it is from the website fetching or the missing chunk detection

Reply:
This should be added in the comment because this review will dispappear...